conans/client/conf/__init__.py

```diff
@@ -11,9 +11,13 @@
 
 MIN_SERVER_COMPATIBLE_VERSION = '0.12.0'
 
-default_settings_yml = """os: [Windows, Linux, Macos, Android, iOS, FreeBSD]
+default_settings_yml = """os: [Windows, Linux, Macos, Android, iOS, FreeBSD, SunOS]
 arch: [x86, x86_64, ppc64le, ppc64, armv6, armv7, armv7hf, armv8]
 compiler:
+    cc:
```

Comment:
maybe rename it to sun-cc? Just so people do not assume that this would be a generic C++ compiler